packages/react/.storybook/Welcome/Welcome.mdx

```diff
@@ -0,0 +1,15 @@
+# Welcome
+
+- [Carbon Website](https://www.carbondesignsystem.com/)
+- [IBM Design Language](https://www.ibm.com/design/language/)
+
+### Other Frameworks
```

Comment:
Can we make this an `h2` as well so they're aren't so many type sizes on a short page.